04_numeronym/sahidakhtar1/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	result := make([]string, len(vals))
+	for i, v := range vals {
+		result[i] = createNumeronym(v)
+	}
+	return result
+}
+
+func createNumeronym(s string) string {
+	trimedString := strings.TrimSpace(s)
```

Comment:
You can trim more using `strings.Trim(s, <charsToTrim>)`.

Reply:
The intent here is if the string contains more than one word then do nothing and return the same string.  So not using `Trim()`

Reply:
[`strings.Trim`](https://golang.org/pkg/strings/#Trim) takes as second parameter a `cutset`:
> Trim returns a slice of the string s with all leading and trailing Unicode code points contained in cutset removed.

So what @anzboi might be suggesting is to use it as `strings.Trim(s, "\n\t ")`